File: Histograma.php

<?php

$numero = 1;
$positivo = "";
$negativo = "";


while($numero != 0){
    $numero = readline("Ingrese numero: ");

    if($numero > 0){
        $positivo = $positivo . "*";

    }elseif($numero < 0){
        $negativo = $negativo . "*";
    }
}

echo "Positivos: $positivo ";
echo "Negativos: $negativo ";

File: Histograma.py

numero = 1
positivo = ""
negativo = ""

while numero != 0:
    numero = int(input("Ingrese numero: "))
    if numero > 0:
        positivo = positivo + "*"
    elif numero < 0:
        negativo = negativo + "*"
    

print("Positivos: ", positivo) 
print("Negativos: ", negativo ) 